app/rest/rest/src/main/java/io/syndesis/rest/v1/handler/metrics/IntegrationMetricsHandler.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.rest.v1.handler.metrics;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.springframework.stereotype.Component;
+import org.springframework.context.annotation.ComponentScan;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.syndesis.model.Kind;
+import io.syndesis.model.metrics.IntegrationMetricsSummary;
+import io.syndesis.rest.metrics.MetricsProvider;
+import io.syndesis.rest.v1.operations.Resource;
+
+@Path("/metrics/integrations")
+@Api(value = "integration-metrics")
+@ComponentScan("io.syndesis.rest.metrics")
```

Comment:
Do we prefer to use auto-configuration in the metrics module or leave ComponentScan here? @zregvart any preference?